src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleKeyUp = (event: KeyboardEvent) => (
+    this.setState({ key: event.key })
+  );
+
+  render() {
+    const { key } = this.state;
+
+    return (
+      <div className="App">
+        {
+          !(key)
+            ? <p className="App__message">Nothing was pressed yet</p>
+            : <p className="App__message">{`The last pressed key is [${key}]`}</p>
+        }
```

Comment:
you repeated the code with adding paragraph two times
let's move the check inside of the paragraph, and will render corresponding message according to the `key` variable